### PR TITLE
Add pvc to chart

### PR DESCRIPTION
**Pull request by TerryHowe**
Two things here:
* Added PVC to test helm chart
* Changed helm chart so it pulls the matching image tag

### CODE REVIEW

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/150#" title="Author self-approved">TerryHowe</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere-build-tooling).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere-build-tooling/blob/main/OWNERS)~~ [TerryHowe]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by abhay-krishna** [comment]
@TerryHowe: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
etcdadm-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/etcdadm-tooling-presubmit/1457834470949261312) | `/test etcdadm-tooling-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-anywhere-build-tooling&pr=150). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:TerryHowe).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by abhay-krishna** [comment]
@TerryHowe: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
etcdadm-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/etcdadm-tooling-presubmit/1457834470949261312) | `/test etcdadm-tooling-presubmit`
vsphere-csi-driver-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/vsphere-csi-driver-tooling-presubmit/1457834481896394752) | `/test vsphere-csi-driver-tooling-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-anywhere-build-tooling&pr=150). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:TerryHowe).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by abhay-krishna** [comment]
@TerryHowe: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
etcdadm-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/etcdadm-tooling-presubmit/1457834470949261312) | `/test etcdadm-tooling-presubmit`
vsphere-csi-driver-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/vsphere-csi-driver-tooling-presubmit/1457834481896394752) | `/test vsphere-csi-driver-tooling-presubmit`
cluster-api-provider-vsphere-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/cluster-api-provider-vsphere-tooling-presubmit/1457834468810166272) | `/test cluster-api-provider-vsphere-tooling-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-anywhere-build-tooling&pr=150). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:TerryHowe).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by abhay-krishna** [comment]
@TerryHowe: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
etcdadm-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/etcdadm-tooling-presubmit/1457834470949261312) | `/test etcdadm-tooling-presubmit`
vsphere-csi-driver-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/vsphere-csi-driver-tooling-presubmit/1457834481896394752) | `/test vsphere-csi-driver-tooling-presubmit`
cluster-api-provider-vsphere-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/cluster-api-provider-vsphere-tooling-presubmit/1457834468810166272) | `/test cluster-api-provider-vsphere-tooling-presubmit`
cloud-provider-vsphere-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/cloud-provider-vsphere-tooling-presubmit/1457834468650782720) | `/test cloud-provider-vsphere-tooling-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-anywhere-build-tooling&pr=150). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:TerryHowe).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by abhay-krishna** [comment]
@TerryHowe: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
etcdadm-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/etcdadm-tooling-presubmit/1457834470949261312) | `/test etcdadm-tooling-presubmit`
vsphere-csi-driver-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/vsphere-csi-driver-tooling-presubmit/1457834481896394752) | `/test vsphere-csi-driver-tooling-presubmit`
cluster-api-provider-vsphere-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/cluster-api-provider-vsphere-tooling-presubmit/1457834468810166272) | `/test cluster-api-provider-vsphere-tooling-presubmit`
cloud-provider-vsphere-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/cloud-provider-vsphere-tooling-presubmit/1457834468650782720) | `/test cloud-provider-vsphere-tooling-presubmit`
kube-vip-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/kube-vip-tooling-presubmit/1457834476859035648) | `/test kube-vip-tooling-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-anywhere-build-tooling&pr=150). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:TerryHowe).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by abhay-krishna** [comment]
@TerryHowe: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
etcdadm-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/etcdadm-tooling-presubmit/1457834470949261312) | `/test etcdadm-tooling-presubmit`
vsphere-csi-driver-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/vsphere-csi-driver-tooling-presubmit/1457834481896394752) | `/test vsphere-csi-driver-tooling-presubmit`
cluster-api-provider-vsphere-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/cluster-api-provider-vsphere-tooling-presubmit/1457834468810166272) | `/test cluster-api-provider-vsphere-tooling-presubmit`
cloud-provider-vsphere-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/cloud-provider-vsphere-tooling-presubmit/1457834468650782720) | `/test cloud-provider-vsphere-tooling-presubmit`
kube-vip-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/kube-vip-tooling-presubmit/1457834476859035648) | `/test kube-vip-tooling-presubmit`
etcdadm-bootstrap-provider-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/etcdadm-bootstrap-provider-tooling-presubmit/1457834469862936576) | `/test etcdadm-bootstrap-provider-tooling-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-anywhere-build-tooling&pr=150). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:TerryHowe).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by abhay-krishna** [comment]
@TerryHowe: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
etcdadm-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/etcdadm-tooling-presubmit/1457834470949261312) | `/test etcdadm-tooling-presubmit`
vsphere-csi-driver-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/vsphere-csi-driver-tooling-presubmit/1457834481896394752) | `/test vsphere-csi-driver-tooling-presubmit`
cluster-api-provider-vsphere-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/cluster-api-provider-vsphere-tooling-presubmit/1457834468810166272) | `/test cluster-api-provider-vsphere-tooling-presubmit`
cloud-provider-vsphere-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/cloud-provider-vsphere-tooling-presubmit/1457834468650782720) | `/test cloud-provider-vsphere-tooling-presubmit`
kube-vip-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/kube-vip-tooling-presubmit/1457834476859035648) | `/test kube-vip-tooling-presubmit`
etcdadm-bootstrap-provider-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/etcdadm-bootstrap-provider-tooling-presubmit/1457834469862936576) | `/test etcdadm-bootstrap-provider-tooling-presubmit`
flux-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/flux-tooling-presubmit/1457834471783927808) | `/test flux-tooling-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-anywhere-build-tooling&pr=150). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:TerryHowe).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by abhay-krishna** [comment]
@TerryHowe: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
etcdadm-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/etcdadm-tooling-presubmit/1457834470949261312) | `/test etcdadm-tooling-presubmit`
vsphere-csi-driver-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/vsphere-csi-driver-tooling-presubmit/1457834481896394752) | `/test vsphere-csi-driver-tooling-presubmit`
cluster-api-provider-vsphere-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/cluster-api-provider-vsphere-tooling-presubmit/1457834468810166272) | `/test cluster-api-provider-vsphere-tooling-presubmit`
cloud-provider-vsphere-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/cloud-provider-vsphere-tooling-presubmit/1457834468650782720) | `/test cloud-provider-vsphere-tooling-presubmit`
kube-vip-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/kube-vip-tooling-presubmit/1457834476859035648) | `/test kube-vip-tooling-presubmit`
etcdadm-bootstrap-provider-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/etcdadm-bootstrap-provider-tooling-presubmit/1457834469862936576) | `/test etcdadm-bootstrap-provider-tooling-presubmit`
flux-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/flux-tooling-presubmit/1457834471783927808) | `/test flux-tooling-presubmit`
etcdadm-controller-tooling-presubmit | 6f250e38fe56a0d4503a5da7933a5b912a91afb1 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/150/etcdadm-controller-tooling-presubmit/1457834470114594816) | `/test etcdadm-controller-tooling-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-anywhere-build-tooling&pr=150). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:TerryHowe).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Review by ewollesen — APPROVED** [review]
Nothing to block a merge, but at least one thing we might want to adjust going forward.

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/150#pullrequestreview-803246408" title="Approved">ewollesen</a>*, *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/150#" title="Author self-approved">TerryHowe</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere-build-tooling).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere-build-tooling/blob/main/OWNERS)~~ [TerryHowe,ewollesen]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: build/lib/helm_build.sh
Comment (TerryHowe) [reply build/lib/helm_build.sh:24]: Line 23 has forces REGISTRY to be passed to the script and line 24 defaults it if it is an empty string.

File: build/lib/helm_build.sh
Comment (TerryHowe) [reply build/lib/helm_build.sh:33]: This is how the script was handling local builds vs push builds

File: projects/aws/eks-anywhere-test/conf/entrypoint.sh
Comment (TerryHowe) [reply projects/aws/eks-anywhere-test/conf/entrypoint.sh:6]:
```suggestion
export pvcsize="$(df --human-readable  --output=size /pvc | tail -1)"
```

Comment (TerryHowe) [reply projects/aws/eks-anywhere-test/conf/entrypoint.sh]: Done

**Pull request merged**
